README.md

```diff
@@ -15,15 +15,41 @@
 
 ### If you want to learn more about these repositories, check out their directories!
 
-### A note to contributers:
-    
- - You don't have to fork! Instead, make your own branch in the central repo. 
+## Contributing to PieCentral
+### Setting up PieCentral
+
+```
+$ cd {directory of your choice}
+$ git clone https://github.com/pioneers/PieCentral.git
+$ cd PieCentral
+```
+
+### Creating a Branch
+Naming convention: project_name/feature_name
+
+Feel free to name "feature_name" anything and/or use more slashes.
+
+e.g. dawn/UDPintegration, atalanta/andy/UDPintegration
+```
+$ git branch {project_name/feature_name}
```

Comment:
Change of branch naming convention LGTM, because it can help resolve cross-project branch confusion.